### PR TITLE
chore(deps): bump clickhouse-server version

### PR DESCRIPTION
**Pull request by cpcloud**
Managed to get backwards compatible code working with the latest clickhouse-server version bump

### CODE REVIEW

File: ibis/backends/sql/compilers/clickhouse.py [hunk 1]
```diff
@@ -758,3 +753,34 @@ def visit_ArraySum(self, op, *, arg):
 
     def visit_ArrayMean(self, op, *, arg):
         return self.f.arrayReduce("avg", self._array_reduction(arg))
+
+    def _promote_bitshift_inputs(self, *, op, left, right):
+        # clickhouse is incredibly pedantic about types allowed in bit shifting
+        #
+        # e.g., a UInt8 cannot be bitshift by more than 8 bits, UInt16 by more
+        # than 16, and so on.
```
Comment (gforsyth) [inline ibis/backends/sql/compilers/clickhouse.py:761]: woof

File: ibis/backends/sql/compilers/clickhouse.py [hunk 2]
```diff
@@ -97,7 +95,6 @@ class ClickHouseCompiler(SQLGlotCompiler):
         ops.Last: "anyLast",
         ops.Ln: "log",
         ops.Log10: "log10",
-        ops.MapContains: "mapContains",
```
Comment (gforsyth) [inline ibis/backends/sql/compilers/clickhouse.py:100]: is this being handled elsewhere?

**Review by gforsyth — APPROVED** [review]
looks good, one non-blocking question about `mapContains`

File: ibis/backends/sql/compilers/clickhouse.py (hunk 2 above)
Comment (cpcloud) [reply ibis/backends/sql/compilers/clickhouse.py:100]: Yep, there's a new `visit_MapContains` implementation near the end of the file.

Comment (gforsyth) [reply ibis/backends/sql/compilers/clickhouse.py:100]: :face_with_peeking_eye:

**Review by gforsyth — APPROVED** [review]

**Pull request merged**